interfaces/sourcegen/sourcegen/_HeaderFileParser.py

```diff
@@ -11,14 +11,15 @@
 
 from ._dataclasses import HeaderFile, Func, Recipe
 from ._helpers import read_config
+from .clib import CLibSourceGenerator
 
 
 _LOGGER = logging.getLogger()
 
 _CLIB_PATH = Path(__file__).parents[3] / "include" / "cantera" / "clib"
 _CLIB_IGNORE = ["clib_defs.h", "ctmatlab.h"]
 
-_DATA_PATH = Path(__file__).parent / "_data"
+_PWD = Path(__file__).parent
```

Comment:
Technically, this is not the present working directory, it's the directory where this file is location. Just suggest a minor rename, I usually prefer something like `HERE`.